LayoutTests/imported/w3c/web-platform-tests/css/css-animations/CSSAnimation-getKeyframes-crash.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
```

Comment:
This doesn't have results/ref. Is that because -crash.html?

Reply:
I believe so, `run-webkit-tests` never yielded any `-expected.txt` file or complained one was missing.